packages/components/src/components/radio-button/radio-button.stories.ts

```diff
@@ -0,0 +1,190 @@
+import '../../solid-components';
```

Comment:
I know the ticket description is not what we want to have but wouldnt it make sense to have a story with multiple buttons to show the functionality. Just by looking at it, I did not directly understand the use-case for this component.

This would also support the question on how to set the border radius to 0 when I have multiple buttons...

Reply:
ah... just found the story in the radio-group...

What is the reason, why we have it there? just to understand it...

Reply:
I also find the presentation a bit strange, Shoelace uses groups in every example:
https://shoelace.style/components/radio-button

Reply:
I hear you, after trying out some different options I think it still makes sense to have examples with just one radio-button so that we can actually show the component and it's controls in the stories (we did the same with sd-radio for example). However I added more examples of button groups (both in sd-radio-button and sd-radio-group) and added more descriptions in documentation so it should all be more clear now I think. Let me know how it looks like to you.

Reply:
Well it technically is part of the sd-radio-group, I later added more examples to the sd-radio-button to make it more clear for users since I got feedback that it's not very intuitive and added different examples with and without labels so it's not repetitive, but I'm open to discuss this more and maybe find a better solution.. 

As for the mouseless story, similar topic, there is already one like that added in the sd-radio-group, so I'm thinking maybe there's no need for the first one to even exist? I see the same case in sd-radio..

Reply:
as discussed in our call:
- remove the group and mouseless story in the `sd-radio-button` component in order to be aligned with the `sd-radio`. 
- add a description to the `sd-radio-button` component which explains that its an equivalent to the `sd-radio` and its basically just a different radio-group-item next to the `sd-radio`.
- add the backgrounds from the current story (Groups and Backgrounds) to another story in the `sd-radio-button` to keep the backgrounds visible in the stories.
- remove or refactor the description of the radio-button-group stories in the `sd-radio-group` component in oder to make clear, that there is no such component as `sd-radio-button-group`

thx